packages/palette/src/elements/Avatar/Avatar.shared.tsx

```diff
@@ -1,14 +1,14 @@
 import React, { ImgHTMLAttributes } from "react"
 import { color } from "../../helpers/color"
 import { styledWrapper } from "../../platform/primitives"
-import { SerifSize } from "../../Theme"
+import { SansSize } from "../../Theme"
 import { Flex } from "../Flex"
-import { Serif } from "../Typography"
+import { Sans } from "../Typography"
 
 export interface SizeProps {
   [key: string]: {
     diameter: string
-    typeSize: SerifSize
+    typeSize: SansSize
```

Comment:
@zephraph one thing I'm not sure about is how we should setup the `Size` mapping for `xs`, `sm`, and `md`. Do we have this documented anywhere?

Reply:
I'm referring to [these](https://github.com/artsy/palette/blob/master/packages/palette/src/elements/Avatar/Avatar.shared.tsx#L16-L29).

Reply:
The type sizes or the avatar image size itself? I assume the type sizes would remain the same. If not, we could probably those from figma... let me see if I can dig that up.

Reply:
The designs aren't very exhaustive at this point. Let's just leave the sizes as is, we can iterate on it if design would like different sizes.